assets/base.css

```diff
@@ -354,73 +354,73 @@ h5,
   font-family: var(--font-heading-family);
   font-style: var(--font-heading-style);
   font-weight: var(--font-heading-weight);
-  letter-spacing: 0.06rem;
+  letter-spacing: calc(var(--font-heading-scale) * 0.06rem);
   color: rgb(var(--color-foreground));
   line-height: 1.3;
 }
```

Comment:
The `login` seems to change size, but I believe this is in an `a` tag https://screenshot.click/28-07-2hzor-qptfj.png

Reply:
Updated. The login link and its associated icon should scale with `body_scale` and **not** `heading_scale` setting. This has been fixed